docker-sandbox/create-workspace.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
```

Comment:
Instead of a shell script, we could just check in the files here?

Reply:
We could, but then you'd have to edit the docker image in two different files :(

Reply:
why would you change the docker image?

Reply:
(Also, we can change Bazel to use the container-image property. That should be pretty quick to get merged.)

Reply:
Done.